CONTRIBUTION.md

```diff
@@ -52,11 +52,7 @@ If you need to integrate images in your entry, add them next to the Markdown fil
 
 Once the page is created you need to add it to the [README.md](README.md) in the right section.
 
-You can add a badge to indicate the progress of the page. 
-
-The current badges we use are:
-- ![Unfinished](https://img.shields.io/badge/Progress-Unfinished-yellow.svg?style=flat) : `![Unfinished](https://img.shields.io/badge/Progress-Unfinished-yellow.svg?style=flat)`  We use it to signal to the user that the page is not yet finished but it already contains useful informations.
-- ![Review](https://img.shields.io/badge/Progress-Review-blue.svg?style=flat) : `![Review](https://img.shields.io/badge/Progress-Review-blue.svg?style=flat)` We use it to signal that the first version of the page is finished but need more review for the content or the quality of the english. 
+> Note: If you have contents to start a page but don't have the time nor the knowledge to finish it, it is advised to propose a pull request with the content you have and to open issues with the missing sections pointing to the concerned page.
```

Comment:
```suggestion
> Note: If you have contents to start a page but don't have the time or the knowledge to finish it, it is advised to propose a pull request with the content you have already. Then, one can open issues with the missing sections pointing to the concerned page.
```

Reply:
I had the impression that `nor` warked in English but I trust you :P

Reply:
The idea is that one can not have the time but have the knowledge or the other way around.
Nor implies that one does not have the time **and** one does not have the knowledge.

Well, if I understand nor correctly :-)